src/World.cpp

```diff
@@ -558,7 +558,7 @@ void cWorld::Start(void)
 	if (!BadWorlds.empty())
 	{
 		const char * WorldName = m_WorldName.c_str();
-		LOGERROR("\n###### ERROR: %s is linked to the following nonexisting world/s:\n%s\n\nPlease edit %s/world.ini and fix this.\n\nNote that the server started enforcing proper world linkages recently. And people with older configs may naturally get this error. If you just want a working default config and don't mind losing this world, delete the folder \"%s\" and the server will receate one for you. Otherwise edit the world.ini file and fix the invalid linkages.\n######\n",
+		LOGERROR("\n###### ERROR: \"%s\" is linked to the following nonexisting world/s:\n%s\n\nPlease edit %s/world.ini and fix this.\n\nNote that the server started enforcing proper world linkages recently. And people with older configs may naturally get this error. If you just want a working default config and don't mind losing this world, delete the folder \"%s\" and the server will receate one for you. Otherwise edit the world.ini file and fix the invalid linkages.\n######\n",
```

Comment:
Why do so many have broken linkage anyway?


On 9 Feb 2016, at 07:27, "LogicParrot" <notifications@github.com> wrote:

> In src/World.cpp:
> 
> > @@ -558,7 +558,7 @@ void cWorld::Start(void)
> >  	if (!BadWorlds.empty())
> >  	{
> >  		const char * WorldName = m_WorldName.c_str();
> > -		LOGERROR("\n###### ERROR: %s is linked to the following nonexisting world/s:\n%s\n\nPlease edit %s/world.ini and fix this.\n\nNote that the server started enforcing proper world linkages recently. And people with older configs may naturally get this error. If you just want a working default config and don't mind losing this world, delete the folder \"%s\" and the server will receate one for you. Otherwise edit the world.ini file and fix the invalid linkages.\n######\n",
> > +		LOGERROR("\n###### ERROR: \"%s\" is linked to the following nonexisting world/s:\n%s\n\nPlease edit %s/world.ini and fix this.\n\nNote that the server started enforcing proper world linkages recently. And people with older configs may naturally get this error. If you just want a working default config and don't mind losing this world, delete the folder \"%s\" and the server will receate one for you. Otherwise edit the world.ini file and fix the invalid linkages.\n######\n",
> I wanted to make it as different as possible because this is an error lots of people are going to get when starting their servers in the next few days, and I didn't want them to confuse it with a crash. The new lines are for tidyness's sake, unlike stacktraces, lots of people will read this. I wanted to explain the issue in detail.
> 
> —
> Reply to this email directly or view it on GitHub.
>